.github/workflows/deploy.yml

```diff
@@ -32,15 +29,21 @@ jobs:
       - name: Install NPM dependencies
         run: npm install
 
```

Comment:
```suggestion
      - name: Install NPM dependencies
        run: npm install

      - name: Build plugin
        run: npm run build
```

Reply:
Yes, this need to be built before it can be pushed up.